packages/marketplace/test/fixtures/exchange.ts

```diff
@@ -0,0 +1,77 @@
+import {ethers, upgrades} from 'hardhat';
+
+import {signerSetup} from './signers';
+import {orderValidatorSetup} from './orderValidator';
+import {royaltiesRegistrySetup} from './royaltiesRegistry';
+import {TrustedForwarderSetup} from './TrustedForwarderMock';
+
+export async function exchangeSetup() {
+  const {admin, user, defaultFeeReceiver} = await signerSetup();
+
+  const {TrustedForwarder} = await TrustedForwarderSetup();
+  const {
+    RoyaltiesRegistryAsUser,
+    Royalties2981ImplMock,
+    RoyaltiesRegistryAsDeployer,
+  } = await royaltiesRegistrySetup();
```

Comment:
You can assign just one variable and then use { ...var, ...var2} in the return clause.